src/main/java/org/jenkinsci/test/acceptance/docker/DockerImage.java

```diff
@@ -12,10 +13,14 @@
  * @author Kohsuke Kawaguchi
  */
 public class DockerImage {
+
+    public static final String DEFAULT_DOCKER_HOST = "127.0.0.1";
     public final String tag;
+    DockerHostResolver dockerHostResolver;
 
     public DockerImage(String tag) {
         this.tag = tag;
+        dockerHostResolver = new DockerHostResolver();
```

Comment:
> should probably use `@Inject` here instead ...

Nooooooooo .... that sh1t is not nicer .... it's putrid ... sorry, I really hate how that crap ends up being used everywhere, just coz it can be used.